src/rpc/misc.cpp

```diff
@@ -1313,12 +1440,13 @@ template <class IndexPair>
 void DecorateEasySendTransactionInformation(UniValue& ret, const IndexPair& pair, const size_t& confirmations)
 {
     const auto& key = pair.first;
-    ret.push_back(Pair("found", true));
+
     ret.push_back(Pair("txid", key.txhash.GetHex()));
     ret.push_back(Pair("index", static_cast<int>(key.index)));
-    ret.push_back(Pair("amount", ValueFromAmount(GetAmount(pair.second))));
+    ret.push_back(Pair("amount", !key.invite ? ValueFromAmount(GetAmount(pair.second)) : GetAmount(pair.second)));
     ret.push_back(Pair("spending", key.spending));
     ret.push_back(Pair("confirmations", static_cast<int>(confirmations)));
+    ret.push_back(Pair("invite", key.invite));
```

Comment:
the value should have if it's an invite, not sure you need invite on the key.

Reply:
same here